### PR TITLE
SRM dashboards'

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by SudheendraKatte** [comment]
@shankaranandh @deepak-harness , please review and provide comments.

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+To view the SRM dashboards, follow these steps:
+
+1. In your Harness project, navigate to **Dashboards**. 
+
+   The Dashboards page is displayed.
+    
+2. Select **Service Reliability**.
+   
+   The dashboard list is displayed, that includes the following pre-configured Service Reliability dashboards:
```
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md:30]:
```suggestion
   The dashboard list is displayed, which includes the following pre-configured Service Reliability dashboards:
```

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Harness Service Reliability Management (SRM) features pre-configured dashboards that offer consolidated insights into the health and historical performance of your Service Level Objectives (SLOs). These dashboards encompass data from all projects and environments within your organization.
+
+Harness dashboards provide you with the ability to:
+
+- Tailor, personalize, and even craft your own dashboards to suit your specific needs.
+	
+- Set up automated delivery schedules and configure alert notifications based on the dashboard data.
+
+For a complete guide to Harness dashboards, go to [Dashboards](https://developer.harness.io/docs/category/dashboards-1).
```
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md:15]:
```suggestion
For a complete guide to Harness dashboards, go to [Dashboards](/docs/category/dashboards-1).
```

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   - SLO History
+
+<docimage path={require('./static/srm-dashboards.png')} />
+
+
+## View a specific SRM dashboard
+
+To view a specific Service Reliability dashboard, do the following:
+
+1.	In the dashboard list, locate the dashboard you wish to explore and select it. The selected dashboard opens, displaying the relevant metrics and data. For example, if you want to view the SLO Health dashboard, simply select it in the list.
```
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md:42]:
```suggestion
- In the dashboard list, locate the dashboard you wish to explore and select it. The selected dashboard opens, displaying the relevant metrics and data. For example, if you want to view the SLO Health dashboard, simply select it in the list.
```

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+- Set up automated delivery schedules and configure alert notifications based on the dashboard data.
+
+For a complete guide to Harness dashboards, go to [Dashboards](https://developer.harness.io/docs/category/dashboards-1).
+
+This topic provides comprehensive insights into Harness SRM dashboards.
+
+
+## View Service Reliability dashboards
+
+To view the SRM dashboards, follow these steps:
```
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md:22]:
```suggestion
To view the SRM dashboards, do the following:
```

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md (hunk 3 above)
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md:42]: I think when there's only one item in a list/steps, we use a bullet

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+The SLO Health dashboard, in its default configuration, provides comprehensive insights into all the SLOs within your Harness organization. 
+
+
+## View SLO Health dashboard
+
+To access the SLO Health dashboard, do the following:
+
+1. Within your Harness project, navigate to **Dashboards**. On the Dashboards page, select **Service Reliability**.
+   
+   The dashboard list is displayed, that includes the default SLO Health and SLO History dashboards.
```
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md:18]:
```suggestion
   The dashboard list is displayed, which includes the default SLO Health and SLO History dashboards.
```

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+   The Dashboards page is displayed.
+    
+2. Select **Service Reliability**.
+   
+   The dashboard list is displayed, that includes the following pre-configured Service Reliability dashboards:
+
+   - SLO Health
+   - SLO History
+
+<docimage path={require('./static/srm-dashboards.png')} />
```
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md:35]:
```suggestion
   <docimage path={require('./static/srm-dashboards.png')} />
```

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+   <docimage path={require('./static/srm-slo-health-dashboard.png')} />
+   
+
+## Explore SLO Health dashboard
+
+SLO Health dashboard consists of the following key components: 
+
+- [SLO Health Status tile](#slo-health-status-tile)
+
+[Health Status Summary tile](#health-status-summary-tile)
```
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md:33]:
```suggestion
- [Health Status Summary tile](#health-status-summary-tile)
```

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+The SLO Health Status tile displays the following information:
+
+- **SLO name**: The name of the SLO.
+
+- **SLO type**: The type or category of the SLO. For example, simple or composite.
+
+- **Evaluation type**: The method used for evaluating the SLO. For example, window or request based.
+
+- **Environment name**: The environment where the SLO is applied. For Example, production, Terraform, and so on.
```
Comment (brian-f-harness) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md:47]:
```suggestion
- **Environment name**: The environment where the SLO is applied. For example, production, Terraform, and so on.
```

**Review by brian-f-harness — APPROVED** [review]
LGTM. Left a few minor suggestions.

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md (hunk 3 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/srm-dashboard/srm-dashboard-intro.md]: Thanks for pointing out!

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+:::info note
+You can customize the data displayed in the SLO Health Status tile using filters to focus on specific aspects of your SLO health metrics.
+:::
+
+
+### Health Status Summary tile
+
+The Health Status Summary tile provides a visual summary of the data presented in the SLO Health Status tile. It includes:
+
```
Comment (Varsha-lalwani) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md:72]: You can add that you can also hover over the  graph and get the count.

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+- **Environment name**: The environment where the SLO is applied. For example, production, Terraform, and so on.
+
+- **Service name**: The name of the service associated with the SLO.
+
+- **Period length**: The duration over which the SLO is measured.
+
+- **Period type**: The type of time period for measurement. For example, calendar or rolling.
+
+- **SLO percentage**: This shows how close you are to achieving your SLO. It's a measure of how well you're meeting your target.
```
Comment (Varsha-lalwani) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md:55]: This is SLI percentage.
There is one more field SLO Target, which is basically SLO target percentage what user defined.

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-history.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+
+- **Environment name**: The environment where the SLO is applied. For example, production, Terraform, and so on.
+
+- **Service name**: The name of the service associated with the SLO.
+
+- **Period length**: The duration over which the SLO is measured.
+
+- **Period type**: The type of time period for measurement. For example, calendar or rolling.
+
+- **SLO percentage**: Measure of how well you're meeting your target. This shows how close you are to achieving your SLO.
```
Comment (Varsha-lalwani) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-history.md:47]: Rename it to SLO Target

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-history.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+- **Total error budget**: The overall error budget allocated for the SLOs, which represents the acceptable margin of error.
+
+- **Target achieved**: Indicates whether the SLO target has been met, denoted as 'Yes' or 'No.' 
+
+- **Remaining error budget**: The remaining portion of the error budget.
+
+:::info note
+You can customize the data displayed in the SLO History tile using filters to focus on specific aspects of your SLO health metrics and history.
+:::
+
```
Comment (Varsha-lalwani) [inline docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-history.md:60]: there is no health status summary tile in history

**Review by Varsha-lalwani — CHANGES_REQUESTED** [review]

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md (hunk 10 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md:55]: @Varsha-lalwani , the SLO Health dashboard doesn't have the SLO Target and SLI Percentage fields. They are part of the SLO History dashboard.

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-history.md (hunk 11 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-history.md]: Addressed

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md (hunk 9 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-health.md:71]: Not documented as the functionality is intuitive.

File: docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-history.md (hunk 12 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/srm-dashboard/srm-dashboard-slo-history.md]: Removed

**Review by Varsha-lalwani — APPROVED** [review]

**Pull request merged**